python/ray/_private/services.py

```diff
@@ -1878,6 +1893,7 @@ def build_java_worker_command(
         plasma_store_name: The name of the plasma store socket to connect
            to.
         raylet_name: The name of the raylet socket to create.
+        redis_username: The username of connect to redis.
```

Comment:
```suggestion
        redis_username: The username to connect to Redis.
```

Reply:
@rynewang, not sure if this rewrite is correct, but the original sentence didn't seem grammatically correct. Please check.

Reply:
your change LGTM